### DISCUSSION TITLE
Add comment of Table or Column Name by OpenAI API

### DISCUSSION THREAD

**Discussion by stella0320** [drawdb-io/drawdb#265] [Ideas]
After designing the structure, I need to type the comments myself. It spent lot of time.....
Because of that, it reminds me a new idea.

If I have a button to trigger, it makes openAI help me automatically generate comment based on colomn name, it's convenient.

My artificial action now is
Step1. Design Sechma on draw.io
<img width="209" alt="image" src="https://github.com/user-attachments/assets/88ec055c-8b35-4c59-8da3-0f39536c7310">

Step2. copy sql script text  into chatGPT and add some prompt after sql script.
`"CREATE TABLE "V_AGENT_GOUP_AUTH_MENU" (
	"ID" INTEGER NOT NULL UNIQUE GENERATED BY DEFAULT AS IDENTITY,
	"AGENT_GROUP_ID" INTEGER NOT NULL,
	"AUTH_MENU_ID" INTEGER NOT NULL,
	PRIMARY KEY("ID")
);

CREATE TABLE "AUTH_MENU" (
	"AUTH_MENU_ID" INTEGER NOT NULL UNIQUE GENERATED BY DEFAULT AS IDENTITY,
	"MENU_NAME" TEXT NOT NULL,
	"URL" TEXT NOT NULL,
	PRIMARY KEY("AUTH_MENU_ID")
);"please filled in by adding comments based on the column name and table name and use English

Step3. ChatGPT help fill up comment based on the column name，and then I copy the comment from chatGPT reponse to my drawdb
<img width="460" alt="image" src="https://github.com/user-attachments/assets/63ff6165-69c4-4a7a-ad98-ae4baaeba38c">

I think the comments given by ChatGPT are almost exactly what I had in mind.

**Comment by 1ilit** [comment]
You can technically use an LLM to generate the whole diagram

**Reply by Str8hurt** [reply]
Mistranslated Translate - Translate WAPI&silicon flowd
//APIesilicon flow free Translate Website, Translate PDFs & Pub eBook,
